platform/ios/src/MGLAnnotationView.h

```diff
@@ -41,10 +41,17 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, assign, getter=isFlat) BOOL flat;
 
 /**
- This property defaults to NO and becomes YES when the view is tapped/clicked on.
+ This property should not be changed directly.
```

Comment:
Thanks, this line should actually be at the end of the comment, rather than the beginning. The first several words of a documentation comment appear in the code completion popup, so it should be descriptive and reasonably unique.

Though it’s intended for Swift and Markdown instead of Objective-C and HeaderDoc, [this Swift API design guideline document](https://swift.org/documentation/api-design-guidelines/#write-doc-comment) has a great guideline on forming documentation comments (scroll up and click “Expand all details now” to see it).